packages/account-v2/src/containers/ProofOfIncome/ProofOfIncomeForm.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { Form, Formik } from 'formik';
+import * as Yup from 'yup';
+import { StandaloneArrowUpFromBracketBoldIcon } from '@deriv/quill-icons';
+import { Button, Divider, Text, useDevice } from '@deriv-com/ui';
+import { FormDropDownField } from '../../components/FormFields';
+import { FormDocumentUploadField } from '../../components/FormFields/FormDocumentUploadField';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { ProofOfIncomeUtils } from '../../utils/proofOfIncomeUtils';
+
+export const ProofOfIncomeForm = () => {
+    const PoincDocumentsList = ProofOfIncomeUtils();
+    const { isMobile } = useDevice();
+    const listItems = [
+        'The document must be up-to-date and signed by the issuance authority.',
+        'The document must contain a letterhead.',
+        'Invalid or incomplete documents shall be rejected.',
+    ];
+    const documentValidation = Yup.mixed().required('Please upload a file');
+
+    return (
+        <div className='relative z-0 h-full w-full  grid gap-16 '>
+            <Formik>
+                <Form>
+                    <section>
+                        <section>
+                            <div className='mb-[2.4rem]'>
+                                <FormSubHeader>Select Document</FormSubHeader>
+                            </div>
```

Comment:
```suggestion
                                <FormSubHeader className='mb-[2.4rem]'>Select Document</FormSubHeader>
```

is it possible to do something like this?

Reply:
on putting className to formsubheader the style doesnot work thats why i has used a div 


![Screenshot 2024-04-26 at 3 27 02 PM](https://github.com/binary-com/deriv-app/assets/158162395/fd4c8c0e-8bfa-4724-996e-9aacdcaebd9b)